aqueductcore/backend/models/extensions.py

```diff
@@ -139,8 +139,9 @@ def execute(
         my_env.update({key: str(val) for key, val in (extension.constants or {}).items()})
         my_env.update(params)
         my_env["aqueduct_url"] = extension.aqueduct_url
-        if extension.aqueduct_key is not None:
-            my_env["aqueduct_key"] = extension.aqueduct_key
+        if extension.aqueduct_api_token is not None:
+            my_env["API_TOKEN"] = extension.aqueduct_api_token
```

Comment:
Is there a strong need in renaming the field?

```suggestion
            my_env["aqueduct_key"] = extension.aqueduct_api_token
```

Reply:
the name of the environment variable that PyAqueduct picks up is `API_TOKEN`. So I used that instead of changing it to key. Also, technically API token and key are different concepts: https://www.gomomento.com/blog/api-keys-vs-tokens-whats-the-difference. What we do is API Token, not key.